src/map/instance.cpp

```diff
@@ -41,25 +41,7 @@ CInstance::CInstance(CZone* zone, uint16 instanceid)
     m_wipeTimer = m_startTime;
 }
 
-CInstance::~CInstance()
-{
-    for (auto entity : m_mobList)
-    {
-        destroy(entity.second);
-    }
-    for (auto entity : m_npcList)
-    {
-        destroy(entity.second);
-    }
-    for (auto entity : m_petList)
-    {
-        destroy(entity.second);
-    }
-    for (auto entity : m_trustList)
-    {
-        destroy(entity.second);
-    }
-}
+CInstance::~CInstance() = default;
```

Comment:
Instead of default, can you slap a Tracy Zone in here, and in the constructor? Then it's good to go 👍

Reply:
Updated, this -- with `TracyZoneScoped;` ?

Reply:
Yessir, looks good 👍